codeclimate-wrapper

```diff
@@ -18,32 +18,52 @@ EOF
   exit 1
 }
 
+docker_for_mac() {
+  test -S /var/tmp/docker.sock > /dev/null 2>&1
+}
+
+socket_exists() {
+  test -S $socket_path > /dev/null 2>&1
+}
+
+socket_path() {
```

Comment:
📝  Docker Toolbox for Mac does still install `boot2docker`. I'll check where it gets installed to.